library/common-library/templates/_labels.tpl

```diff
@@ -16,6 +16,12 @@ helm.sh/chart: {{ include "common.naming.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.global.labels }}
```

Comment:
After visiting the `priorityClass` PR I have to reject this PR because I overlooked this.

We cannot trust that global exists.

Reply:
Do you mean that Values.global can be nil, right?

Reply:
I'll fix that and check why tests were not covering it